plugins/obs-ffmpeg/cmake/legacy.cmake

```diff
@@ -112,6 +112,13 @@ elseif(OS_POSIX AND NOT OS_MACOS)
   find_package(Libpci REQUIRED)
   target_sources(obs-ffmpeg PRIVATE obs-ffmpeg-vaapi.c vaapi-utils.c vaapi-utils.h)
   target_link_libraries(obs-ffmpeg PRIVATE Libva::va Libva::drm LIBPCI::LIBPCI)
+
+  find_package(FFnvcodec 12)
```

Comment:
I think it would be good to have a default-on cmake flag like `ENABLE_NATIVE_NVENC`, as I've had to catch myself multiple times after assuming ffnvcodec was found, when it actually was not.

Reply:
Could be done, yeah, then you could mark it `REQUIRED` and CMake will error out if it's not found. I need to update this to `12.1` anyway.